honeybee_radiance_folder/folder.py

```diff
@@ -599,6 +623,47 @@ def combined_receivers(self, folder='receiver', auto_mtx_path=False):
 
         return receivers_info
 
+    def two_phase_octree_mapping(self):
+        """List of rad files for each state for aperture groups without transmission
+        matrix. These files can be used to create the octree for each specific state."""
+
+        two_phase_mapping = []
+        # add default 2 phase files
+        # TODO: we should probably check if 2 phase is possible, i.e., check if there are
+        #       static apertures and non-BSDF aperture groups.
+        two_phase_mapping.append(
+            {
+                'identifier': '2_phase_default',
```

Comment:
I like the name `__static_apertures__` a bit more since it is about the model and not the simulation technique.